packages/core/src/utils.ts

```diff
@@ -75,6 +75,7 @@ export function toStateValue(
   return pathToStateValue(statePath);
 }
 
+/** @internal */
```

Comment:
Similarly here - this is **explicitly** exported today. It's not like it got exported accidentally through a wildcard or something. I think we should keep it or remove the export itself (from `src/index.ts`).

Reply:
This only removes it from the generated documentation.